app/assets/stylesheets/application/components/footer.scss

```diff
@@ -1,4 +1,5 @@
 footer {
+  display: none; //small screens
```

Comment:
Why don't we do footers on mobile?

Reply:
In the design, the nav is going to be fixed at the bottom of the screen.